.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/VectorStoreTests.cs

```diff
@@ -140,14 +146,16 @@ public async Task CanEnumerateVectorStores()
     public async Task CanAssociateFiles()
     {
         VectorStoreClient client = GetTestClient();
-        VectorStore vectorStore = await client.CreateVectorStoreAsync();
+        CreateVectorStoreOperation createVectorStoreOperation = await client.CreateVectorStoreAsync(false);
+        VectorStore vectorStore = createVectorStoreOperation.GetVectorStore();
         Validate(vectorStore);
 
         IReadOnlyList<OpenAIFileInfo> files = await GetNewTestFilesAsync(client.GetConfigOrThrow(), 3);
 
         foreach (OpenAIFileInfo file in files)
         {
-            VectorStoreFileAssociation association = await client.AddFileToVectorStoreAsync(vectorStore, file);
+            AddFileToVectorStoreOperation addFileToVectorStoreOperation = await client.AddFileToVectorStoreAsync(vectorStore, file, false);
+            VectorStoreFileAssociation association = addFileToVectorStoreOperation.GetFileAssociation();
```

Comment:
Same comment about getting this from the LRO `Value` property.

Reply:
Same reply as the other one